editor.planx.uk/src/ui/shared/ErrorWrapper.tsx

```diff
@@ -18,8 +18,8 @@ export interface Props {
 const Root = styled(Box, {
   shouldForwardProp: (prop) => prop !== "error",
 })<BoxProps & { error: Props["error"] }>(({ theme, error }) => ({
+  width: "100%",
```

Comment:
This took me way to long to get to the bottom of 😅 

Without this change, when we toggle to an @ variable, the input it offset by a tiny amount - 

<img width="620" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/4ed518df-a81d-4f2d-94cd-1a71c72cf431">